### PR TITLE
feat(python): add CAN messages to send limit switch status

### PR DESCRIPTION
**Pull request by caila-marashaj**
adding message definition to get limit switch status via CAN

### CODE REVIEW

File: python/opentrons_ot3_firmware/messages/payloads.py [hunk 1]
```diff
@@ -203,3 +203,10 @@ class FirmwareUpdateCompleteAcknowledge(utils.BinarySerializable):
     """A response to the FirmwareUpdateComplete message."""
 
     error_code: utils.UInt16Field
+
+
+@dataclass
+class GetLimitSwitchResponse(utils.BinarySerializable):
+    """A response to the Limit Swith Status request payload"""
+
```
Comment (DerekMaggio) [inline python/opentrons_ot3_firmware/messages/payloads.py:229]: Do we have a definition of the switch status?
I am assuming an Enum that maps to the int?

File: python/opentrons_ot3_firmware/scripts/generate_header.py [hunk 2]
```diff
@@ -84,7 +84,7 @@ def generate_c(output: io.StringIO) -> None:
 
 def write_enum_c(e: Type[Enum], output: io.StringIO) -> None:
     """Generate constants from enumeration."""
-    output.write(f"/** {e.__doc__} */\n")
+    output.write(f"/** {e.__doc__} */\n"
```
Comment (DerekMaggio) [inline python/opentrons_ot3_firmware/scripts/generate_header.py:87]: Missing a parentheses no?

**Review by DerekMaggio — CHANGES_REQUESTED** [review]
Everything looks good. Except maybe you a missing a paren. Left a comment about it pointing it out.

**Review by DerekMaggio — APPROVED** [review]

File: Pipfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
+[[source]]
```
Comment (sfoster1) [inline Pipfile:1]: This shouldn't be here - happens if you run `pipenv` in the root of the repo accidentally. We should remove this and add `./Pipfile` to `.gitignore`

**Review by sfoster1 — APPROVED** [review]
Let's get this pipfile out of there but it looks good to me otherwise. I think we may end up wanting to allow read limit switch response to contain values for multiple switches but then again we already have the patterns of nodes with multiple axes under control having different addresses for each.

File: Pipfile (hunk 3 above)
Comment (amitlissack) [inline Pipfile:1]: I don't think this file should be included.

**Review by amitlissack — APPROVED** [review]
Please just remove that Pipfile. 

Otherwise, congrats on first PR!

File: python/opentrons_ot3_firmware/messages/message_definitions.py [hunk 4]
```diff
@@ -276,11 +276,10 @@ class FirmwareUpdateCompleteAcknowledge:  # noqa: D101
 class ReadLimitSwitchRequest:  # noqa: D101
     payload: payloads.EmptyPayload
     payload_type: Type[BinarySerializable] = payloads.EmptyPayload
-    message_id: MessageId.limit_sw_request
-
+    message_id: Literal[MessageId.limit_sw_request] = MessageId.limit_sw_request
```
Comment (amitlissack) [inline python/opentrons_ot3_firmware/messages/message_definitions.py:279]: Sorry I missed this in the initial review. 

The significance of the `Literal[MessageId.limit_sw_request]` is to tell our type checker that `message_id` MUST be the value `MessageId.limit_sw_request`.

**Pull request merged**